### PR TITLE
London Class 7 - Oksana Kostenko - Git - Week 1

### PR DESCRIPTION
**Pull request by oksygenn**
# Your Details

- Your Name: Oksana Kostenko
- Your City: London
- Your Slack Name:  Oksana Kostenko

# Homework Details

- Module: Git
- Week: 1

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -4,77 +4,78 @@
...
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website prettier
```
Comment (kmh256) [inline homework.md:31]: a little more description

**Review by kmh256 — COMMENTED** [review]
Amazing Oksana 👍 maybe just a little more description for Question 5. Great work!

**Comment by oksygenn** [comment]
> Amazing Oksana 👍 maybe just a little more description for Question 5. Great work!
Thank you for the review! I added more description for question 5.

**Comment by ademalagoz** [comment]
Well done Oksana

**Pull request closed without merging**